CMakeLists.txt

```diff
@@ -112,8 +115,7 @@ if (BUILD_SUITE)
 	else (PLX_FOUND OR PXI_FOUND)
 		#Find the PLX Library
 	   find_package(PLX REQUIRED)
-   	link_directories(${PLX_LIBRARY_DIR})
-	   #add_definitions("-DPLX_LINUX -DPCI_CODE -DPLX_LITTLE_ENDIAN")
```

Comment:
Yes it was commented out because we were not sure of its function. I don't want to throw it away unless we know we don't need it.